packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -664,6 +666,90 @@ export const largeDelay = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.largeDelay)
 ).withDefault(LargeDelay);
 
+// Banner Tokens
+export const bannerGapSize = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerGapSize)
+).withDefault('1px');
+
+export const bannerButtonHeight = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerButtonHeight)
+).withDefault('24px');
+
+export const bannerFontSize = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerFontSize)
+).withDefault('12.8px');
+
+export const bannerColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerColor)
+).withDefault(White);
+
+export const bannerNeutralBackgroundColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerNeutralBackgroundColor)
+).withDefault((element: HTMLElement) => getColorForTheme(element, Black75, Black75, hexToRgbaCssColor(White, 0.3)));
+
+export const bannerErrorBackgroundColor = DesignToken.create<string>(
+    styleNameFromTokenName(tokenNames.bannerErrorBackgroundColor)
+).withDefault((element: HTMLElement) => getColorForTheme(
+    element,
+    Fail100LightUi,
+    BannerFail100DarkUi,
+    hexToRgbaCssColor(White, 0.3)
```

Comment:
Double checking: is it intentional that all the severities have the same background color in the color theme?

Reply:
Yep
![image](https://user-images.githubusercontent.com/7282195/217597069-3b70eb95-cea4-4330-85eb-3c039ba7c2d8.png)